pkg/build/strategies/sti/postexecutorstep.go

```diff
@@ -218,6 +224,11 @@ func (step *downloadFilesFromBuilderImageStep) downloadAndExtractFile(artifactPa
 			utilstatus.ReasonFSOperationFailed,
 			utilstatus.ReasonMessageFSOperationFailed,
 		)
+		step.builder.result.BuildInfo.StepInfo = utilstatus.AddStepInfo(
+			step.builder.result.BuildInfo.StepInfo,
+			api.GatherInputContentStep,
+			timeStart,
+		)
```

Comment:
this is, i think, part of the extended-build support so we don't need to worry about timing for it.